src/Orc.NuGetExplorer.Xaml/ViewModels/PackageSourceSettingViewModel.cs

```diff
@@ -89,6 +96,11 @@ public PackageSourceSettingViewModel(INuGetConfigurationService configurationSer
 
         private void OnRemoveFeedExecute()
         {
+            if (SelectedFeed is null)
```

Comment:
store in variable. Always good for multithreaded access, but also micro optimization because the code doesn't have to call the getter multiple times.